src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -743,8 +925,19 @@ viewSelectedVerifiers ({ shared } as loggedIn) profileSummary selectedVerifiers
                     (\p ->
                         div
                             [ class "flex justify-between flex-col m-3 items-center" ]
-                            [ Profile.Summary.view shared loggedIn.accountName p profileSummary
-                                |> Html.map GotProfileSummaryMsg
+                            -- We need `absolute` so we can display the dialog
+                            -- bubble on hover
+                            [ div [ class "absolute" ]
+                                [ Profile.Summary.view shared loggedIn.accountName p profileSummary
+                                    |> Html.map GotProfileSummaryMsg
+                                ]
+
+                            -- We need this invisible Profile.Summary so we know
+                            -- the exact space the above one would take
+                            , div [ class "opacity-0 pointer-events-none" ]
+                                [ Profile.Summary.view shared loggedIn.accountName p profileSummary
+                                    |> Html.map GotProfileSummaryMsg
+                                ]
```

Comment:
@victorgdev this is the gambiarra I was talking about. Note that on the [parent](https://github.com/cambiatus/frontend/pull/570/files#diff-05859dfc08b749f5df9f960f031f861fa38f5b07bd54c29d40769d21b61a7b36R324-R329) I added `overflow-y-hidden`, because it wouldn't work if the user was able to scroll the modal